R/query-join.R

```diff
@@ -284,7 +284,8 @@ sql_join_tbls <- function(con, by, na_matches = "never") {
         sql_expr_matches(sql(lhs[[i]]), sql(rhs[[i]]), con = con)
       })
     } else {
-      compare <- paste0(lhs, " = ", rhs)
+      by$condition[by$condition == "=="] <- "="
```

Comment:
This is the only real change to the SQL generation right? Pretty much everything else is improved input validation and new handling of output columns?